app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -263,6 +263,12 @@ def parse_date(year:, month:, day:)
           if year.to_i.positive? && month.to_i.positive? && day.to_i.positive?
             Date.new(year.to_i, month.to_i, day.to_i).to_s
           end
```

Comment:
Minor: If we're now rescuing the exception of an invalid date initialization, should we bother to do any confidence checks on those values?

```suggestion
          Date.new(year.to_i, month.to_i, day.to_i).to_s
```

Reply:
Nice catch, although a bad year could end up returning year 0 which wouldn't throw an exception:
```
Date.new(nil.to_i, 1, 1)
=> Thu, 01 Jan 0000

Date.new('a'.to_i, 1, 1)
=> Thu, 01 Jan 0000
```
I removed the check on month/day in b484da001.